docs/tutorial/02-draw-a-snake.md

```diff
@@ -0,0 +1,244 @@
+### 2. Draw a snake
+
+In our retro game, all objects and their movements are organized in big square "pixels". The pallet is a single pixel and the snake a line of these pixels moving one pixel a step. We'll therefore describe our playing field as a grid of square cells addressable like a coordinate system:
```

Comment:
Yes, I also realized that already... I used "pixel" here as I thought it would be easier to understand for people. But you're right, it's probably more confusing than helpful. Will update.